extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -227,6 +231,10 @@ export class SubscriptionService {
    */
   resolveEntitlementsToStore_(serviceId, entitlement) {
     this.platformStore_.resolveEntitlement(serviceId, entitlement);
+    if (entitlement.decryptedDocumentKey) {
+      this.cryptoHandler_.tryToDecryptDocument(
```

Comment:
Call to this method on a platform returning a valid `decryptedDocumentKey` should be tested.

Reply:
done